native/e2e/recorders/ios-recorder.js

```diff
@@ -15,17 +15,25 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-const { spawn } = require('child_process');
+const { command, rawCommand } = require('../utils');
 
 const recordSimulator = fileName =>
-  spawn('xcrun', [
+  rawCommand(
+    'xcrun',
     'simctl',
     'io',
     'booted',
     'recordVideo',
     '--type=fmp4',
     fileName,
-  ]);
+  );
+
+const optimiseRecordedVideo = filename => {
+  command(`scripts/video/convert-to-mp4.sh`, filename).catch(error => {
```

Comment:
Shoudl we exit with a non-zero status code if this happens too?

Reply:
I dunno - when the tests fail currently we don't do this and the errors still get surfaced, so I'm not sure what difference it would make.

Reply:
I don't think we should as exiting here might cause a problem with the detox process hanging

Reply:
So then the only way to know the process has failed is to look at the output?

Reply:
Maybe I'm missing the point here, but if we're running this stuff manually, we're going to be looking at the output anyway right?

Reply:
No you're right someone will be looking at it

Reply:
Ok cool - is this ok to merge then?

Reply:
:shipit: